utils/ui_helpers.py

```
@@ -0,0 +1,71 @@
+            background-color: {hover_color};
+        }}
+        QPushButton:pressed {{
+            background-color: {pressed_color};
+        }}
+        QPushButton:disabled {{
+            background-color: #bdc3c7;
+            color: #7f8c8d;
+        }}
+    """
+
+def show_status(self, message, status_type="info"):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)